.gitattributes

```diff
@@ -1 +1,2 @@
 numba_scipy/_version.py export-subst
+numba_scipy/special/signatures.py binary
```

Comment:
@person142 -- do you happen to have any idea why this change was made? We are currently proposing to remove it again in #54 because we don't see a need for it and can't see why it would be necessary.

Reply:
No idea. Probably no Chesterton’s fence here.